front/src/core/routes.ts

```diff
@@ -1,5 +1,7 @@
 import { generatePath } from "react-router";
 
+export const baseRoute: String = "http://localhost:3000/api/registry";
```

Comment:
In routes we are covering React Router routes, maybe we should move this to another file, to avoid missconceptions